src/cpp/session/modules/SessionRCompletions.R

```diff
@@ -3323,6 +3323,7 @@ assign(x = ".rs.acCompletionTypes",
    .rs.makeCompletions(
       token = token,
       results = results.sorted,
-      packages = packages.sorted
+      packages = packages.sorted,
+      type = attr(results, "type")
```

Comment:
Actually that brings up the question on if the [scoping / sorting code](https://github.com/jimhester/rstudio/blob/6210df4ffee959fee85e71e3c220c9f765df052b/src/cpp/session/modules/SessionRCompletions.R#L3312-L3321) should be in this function at all. It assumes that the completions will be of function or argument names, which is not the case in any of the custom completions I have implemented thus far.

Perhaps we should get more of the arguments from attributes of the result with defaults if the attributes do not exist.